src/docs/product/performance/web-vitals.mdx

```diff
@@ -71,6 +73,48 @@ Some Web Vitals such as FP, FCP, LCP, and TTFB are measured relative to the star
+
+At the center of the page, Web Vital meters show you the raw vital values for your application and the individual score for each Web Vital. These meters can be useful for prioritizing which Web Vitals need attention most. Click a meter to open a summary of the Web Vital and see which of your pages have the most [Opportunity](#opportunity) for improvement.
+
+At the bottom of the Web Vitals page, a sortable table shows a lists your application's pages, along with their associated P75 values for each Web Vital. Each page also possess its own individual Performance Score. The [Opportunity](#opportunity) column displays a page score's potential impact to your application's overall Performance Score if maximized to 100. The search bar above this table allows users to narrow down to specific pages by name.
+
+You can click through any of your application's pages from the Web Vitals page to get to the [Page Overview](#page-overview) display.
+
+## Page Overview
+
+The **Page Overview** page gives you an overview of a single page of your application. In the **Page Overview**, you can further drill down to a specific page load sample [Event](/product/sentry-basics/concepts/tracing/event-detail/), [Replay](/product/session-replay/), or [Profile](/product/profiling/). You can navigate to the **Page Overview** of any of your application's pages through the main table in the [Web Vitals](#web-vitals-page) page, or through the slideout panel when clicking on a Web Vital meter in the [Web Vitals](#web-vitals-page) page.
+
+The **Page Overview** page displays a series of "Throughput", "P75 Duration", "5XX Responses" charts in the right sidebar of the page. A mini **Aggregate Span Waterfall** is also displayed in the right sidebar, which shows you common span paths that your application's page may take. Click the "View Full Waterfall" button or the "Aggregate Spans" tab at the top of the page to see the full **Aggregate Span Waterfall**.
+
+At the center of the **Page Overview**, Web Vital meters and suspect tags are displayed. Clicking a Web Vital meter will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Event Detail](/product/sentry-basics/concepts/tracing/event-detail/) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
```

Comment:
meters feels like a weird word to use here. I would suggest changing it to metrics or value or score, unless there's a reason to say "meter"?

Reply:
No particular reason to use "meter", i've updated to use "score" instead!